client/components/tooltip/style.scss

```diff
@@ -196,4 +200,45 @@
 			background: var(--studio-gray-100);
 		}
 	}
+
+	&.post-trends-day__tooltip {
```

Comment:
Do you think this is a better place? We are putting a specific CSS class (for one component) in a generic component. 
I had a similar problem with the annual highlight popover. If we need the same styles, maybe it would be better to create another variant for the tooltip (and avoid components specific classes in favour of something like a tooltip--dark)?

Reply:
What did you do in this situation? Can I take a look at your PR?

Reply:
I kept most of the CSS specific to my component in that component's SCSS file. That's what you did initially here.
Here's the PR: https://github.com/Automattic/wp-calypso/pull/71159/files 
I think we share some of the styles, so maybe we should extract them into yet another component.
If you've got a different idea, we can talk about it, but if you'd like to follow this one, it might be better to do it in another PR. What do you think?

Reply:
@grzegorz-cp got it. I like this idea. I'll create the variant for the tooltip on this PR, and regarding extracting them into another component, I agree that we can do it in another PR.